composer.json

```diff
@@ -91,7 +91,7 @@
             "@ci:static",
             "@ci:dynamic"
         ],
-        "phive:update:phpunit": "echo y | \"./tools/phive.phar\" update phpunit"
+        "phive:update:phpunit": "\"./tools/phive.phar\" update phpunit --trust-gpg-keys --no-progress"
```

Comment:
If I read the Phive documentation correctly, we'll probably need to specify a comma-separated list of key IDs:

```
--trust-gpg-keys         Silently import these keys when required (separated by comma)
```

Let's see whether this works without the key IDs as well in the build.